fml/thread.h

```diff
@@ -28,6 +74,8 @@ class Thread {
 
  private:
   std::unique_ptr<std::thread> thread_;
+  /// ThreadConfigure is used for setting thread configure some like `priority`
+  std::unique_ptr<ThreadConfig> thread_config_;
```

Comment:
There isn't a need to hold onto this variable is there?

Reply:
The ThreadConfig set Thread is in the Thread callback, If don't hold it , it may be released when use it.

Reply:
But it's only used in the constructor.

Reply:
(the lambda can just take ownership of it)